src/views/MainPage/MainPage.js

```diff
@@ -0,0 +1,49 @@
+import React, {useEffect, useState} from "react";
+import TopSection from "../TopSection/TopSection";
+import HotelsResultSection from "../HotelsResultSection/HotelsResultSection";
+import Footer from "../Footer/Footer";
+import {useSelector} from "react-redux";
+import {Navigate} from "react-router-dom";
+
+function MainPage() {
+    const [searchValue, setSearchValue] = useState([])
+    const [serverData, setServerData] = useState([])
+    const [value, setValue] = useState('0')
+
+    useEffect(() => {
+
+        fetch('https://fe-student-api.herokuapp.com/api/hotels/popular')
+            .then((response) => {
+                return response.json()
+            })
+            .then(setServerData)
+    }, [])
+
+
+    useEffect(() => {
+
+        fetch(`https://fe-student-api.herokuapp.com/api/hotels?search=${value}`)
+            .then((response) => {
+                return response.json()
+            })
+            .then(setSearchValue)
```

Comment:
does it work? Im not sure case you don't call this function

Reply:
it works)